lte/gateway/python/magma/kernsnoopd/snooper.py

```diff
@@ -0,0 +1,107 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+import os
+
+from bcc import BPF
+from jinja2 import Template
+from magma.common.job import Job
+
+from magma.kernsnoopd.handlers import ebpf_handlers
+
+# TODO: what's the right way to look for these files?
+EBPF_SRC_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'ebpf')
```

Comment:
Will non-Python files be here when Magma is packaged? Beside the Python source file?

Reply:
cc @tmdzk / @quentinDERORY 

I suspect you'll have to handle them in build-magma.sh / post install but will let the CI folks give their expertise

Reply:
It looks like I should use `glob_files()` in `build-magma.sh` to place my static files in `/var/opt/magma/ebpf/`, and directly read from absolute path in the service. Does that sound right? @tmdzk @quentinDERORY